### DISCUSSION TITLE
[Question]: What should I do if I just want client side code?

### DISCUSSION THREAD

**Discussion by cruldra** [danny-avila/LibreChat#2292] [Q&A]
### What is your question?

What should I do if I just want client side code

### More Details

I apologize if this question offended you, but what I'm trying to say is, am I allowed to use only client code? I tried opening the client directory as a brand new project, but it didn't work.

Thanks!
Also, I was wondering why not put each module in its own separate repository

### What is the main subject of your question?

UI

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
This is a full stack app, it is deeply coupled to the backend, without which, it cannot run properly. There are many ways to develop over it, even focusing exclusively on the frontend**, but it relies on API interactions with the backend.

**Reply by cruldra** [reply]
Can I separate out the client and provide the backend myself?

**Reply by danny-avila** [reply]
You can’t, not without some pretty involved refactoring
